include/boost/beast/websocket/stream.hpp

```diff
@@ -38,6 +38,30 @@
 namespace boost {
 namespace beast {
 namespace websocket {
+/** permessage-deflate extension status.
+
+    These settings indicate the status of the permessage-deflate
+    extension, showing if it is active and the window bits in use.
+
+    @note
+
+    Used to check the status of the permessage-deflate extension after
+    the WebSocket handshake.
```

Comment:
Please move this note to the Javadoc of `stream::get_status`. Please squash all the commits in a single commit with a message like 'Make permessage-deflate status queryable in websocket::stream'.